src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const sexPartner = (person.sex === 'm') ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      <p className="Person__age">{person.age && `I am ${person.age}`}</p>
```

Comment:
If there is no age then paragraph will be still rendered
```suggestion
     {age &&  <p className="Person__age">`I am ${age}`</p>}
```